packages/fns/castArray.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Casts value to array
+ *
+ * @param {any} value
+ * @returns {Array} casted value
+ */
+const castArray = value => {
+  return Array.isArray(value) ? value : [value];
+};
+
+export default castArray;
```

Comment:
Right now this `fns` dir (is there a better name? I like to keep it short and simple) isn't technically a package, and each package importing from here will include it's own copy of this helpers. That's a small price to pay. I plan to later convert it into our own set of "lodash" package